homeassistant/components/mqtt/quality_scale.yaml

```diff
@@ -86,7 +86,10 @@ rules:
     comment: >
       This is not possible because the integrations generates entities
       based on a user supplied config or discovery.
-  reconfiguration-flow: done
+  reconfiguration-flow:
+    status: exempt
+    comment: >
+      This integration is reconfigured via options flow.
```

Comment:
I think it's better to mark this as `exempt` rather than `done` as technically it doesn't really have a reconfiguration flow.

Reply:
right, seems better indeed